spec/rails_helper.rb

```diff
@@ -36,7 +36,8 @@
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
-
+  config.file_fixture_path = Rails.root.join('spec', 'support', 'fixtures')
+  config.include MockHelper
```

Comment:
I'm not sure about including that MockHelper here, because it will be loaded for each test and if you have to test an API that is big enough you will have a file with thousands of lines with all the stubs you need. I'd prefer to separate this per API or per endpoint and including it directly in each test file.